shell/browser/ui/devtools_manager_delegate.cc

```diff
@@ -139,4 +142,8 @@ bool DevToolsManagerDelegate::HasBundledFrontendResources() {
   return true;
 }
 
+content::BrowserContext* DevToolsManagerDelegate::GetDefaultBrowserContext() {
+  return browser_context_;
```

Comment:
Instead of plumbing the default session all the way to here, perhaps just 
```suggestion
  return ElectronBrowserContext::From("", false);
```
?